<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_osc1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_osc2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="verticalSlider_oscRatio">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Osc. 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Osc. 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Osc. 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Osc. 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_oscRatio">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.50</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_LFO">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LFO">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5.0 Hz</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_attack">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_decay">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_sustain">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_release">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_filter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High Pass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resonant</string>
     </property>
    </item>
   </widget>
   <widget class="QDial" name="dial_res">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_cutoff">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>440</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cutoff Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cutoff">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>440 Hz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filter">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LFO">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_midi">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Midi Input Device</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resonance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_LFO">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Frequency</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ratio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Distortion</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_attack">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_decay">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1s</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sustain">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_release">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01s</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_LFO_amount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LFO_amount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>50 cents</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_dist">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Overdrive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bitcrusher</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="horizontalSlider_dist_amount">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>224</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_dist_amount">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dist">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>LFO</string>
    </property>
    <widget class="QComboBox" name="comboBox_LFO_wave">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>104</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saw</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wave Form</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Oscillators</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Amplitude Envelope</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Distortion</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>41</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Gain</string>
    </property>
    <widget class="QSlider" name="verticalSlider_gain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>22</width>
       <height>160</height>
      </rect>
     </property>
     <property name="value">
      <number>75</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch Bend Amount</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_pbend">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_pbend">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>100 cents</string>
    </property>
   </widget>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>comboBox_osc1</zorder>
   <zorder>comboBox_osc2</zorder>
   <zorder>verticalSlider_oscRatio</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_oscRatio</zorder>
   <zorder>horizontalSlider_LFO</zorder>
   <zorder>label_6</zorder>
   <zorder>label_LFO</zorder>
   <zorder>verticalSlider_attack</zorder>
   <zorder>verticalSlider_decay</zorder>
   <zorder>verticalSlider_sustain</zorder>
   <zorder>verticalSlider_release</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>comboBox_filter</zorder>
   <zorder>dial_res</zorder>
   <zorder>horizontalSlider_cutoff</zorder>
   <zorder>label_15</zorder>
   <zorder>label_cutoff</zorder>
   <zorder>pushButton_filter</zorder>
   <zorder>pushButton_LFO</zorder>
   <zorder>comboBox_midi</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>comboBox_LFO</zorder>
   <zorder>label_attack</zorder>
   <zorder>label_decay</zorder>
   <zorder>label_sustain</zorder>
   <zorder>label_release</zorder>
   <zorder>horizontalSlider_LFO_amount</zorder>
   <zorder>label_19</zorder>
   <zorder>label_LFO_amount</zorder>
   <zorder>comboBox_dist</zorder>
   <zorder>horizontalSlider_dist_amount</zorder>
   <zorder>label_dist_amount</zorder>
   <zorder>pushButton_dist</zorder>
   <zorder>label_7</zorder>
   <zorder>horizontalSlider_pbend</zorder>
   <zorder>label_pbend</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="actionCrunchy_Bass"/>
    <addaction name="actionDub_Bass"/>
    <addaction name="actionLevel_Up"/>
    <addaction name="actionSmooth_Lead"/>
    <addaction name="actionBusy_Bee"/>
    <addaction name="actionMotor"/>
   </widget>
   <addaction name="menuPresets"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCrunchy_Bass">
   <property name="text">
    <string>Crunchy Bass</string>
   </property>
  </action>
  <action name="actionLevel_Up">
   <property name="text">
    <string>Level Up</string>
   </property>
  </action>
  <action name="actionDub_Bass">
   <property name="text">
    <string>Dub Bass</string>
   </property>
  </action>
  <action name="actionSmooth_Lead">
   <property name="text">
    <string>Smooth Lead</string>
   </property>
  </action>
  <action name="actionBusy_Bee">
   <property name="text">
    <string>Busy Bee</string>
   </property>
  </action>
  <action name="actionMotor">
   <property name="text">
    <string>Motor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox_osc1</tabstop>
  <tabstop>comboBox_osc2</tabstop>
  <tabstop>comboBox_midi</tabstop>
  <tabstop>horizontalSlider_LFO_amount</tabstop>
  <tabstop>horizontalSlider_LFO</tabstop>
  <tabstop>pushButton_LFO</tabstop>
  <tabstop>comboBox_LFO</tabstop>
  <tabstop>verticalSlider_oscRatio</tabstop>
  <tabstop>verticalSlider_attack</tabstop>
  <tabstop>verticalSlider_decay</tabstop>
  <tabstop>verticalSlider_sustain</tabstop>
  <tabstop>verticalSlider_release</tabstop>
  <tabstop>pushButton_filter</tabstop>
  <tabstop>comboBox_filter</tabstop>
  <tabstop>horizontalSlider_cutoff</tabstop>
  <tabstop>dial_res</tabstop>
  <tabstop>horizontalSlider_dist_amount</tabstop>
  <tabstop>comboBox_dist</tabstop>
  <tabstop>pushButton_dist</tabstop>
  <tabstop>verticalSlider_gain</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
